disco_aws_automation/disco_autoscale.py

```diff
@@ -27,9 +28,9 @@ def __init__(self, environment_name, autoscaling_connection=None, boto3_autoscal
         )
         self.boto3_autoscale = boto3_autoscaling_connection or boto3.client('autoscaling')
 
-    def get_groupname(self, hostclass):
-        '''Returns the autoscaling group name when given a hostclass'''
-        return self.environment_name + '_' + hostclass
+    def get_new_groupname(self, hostclass):
+        '''Returns a new autoscaling group name when given a hostclass'''
+        return self.environment_name + '_' + hostclass + "_" + str(int(time.time()))
```

Comment:
hmmm i see you're using it to sort by name/time below.... maybe use `datetime.now()` (which gives better precision) and convert that to millisec or (microsec if you'd like).